libs/design/src/atoms/form/form-field/form-field/form-field.component.ts

```diff
@@ -58,6 +58,15 @@ export class DaffFormFieldComponent implements DoCheck, AfterContentInit, AfterC
    */
   isValid = false;
 
+  /**
+   * @docs
+   *
+   * Determines whether or not the form field should display its focused state.
+   */
+  get isFocused() {
+    return this._control.focused;
```

Comment:
This needs optional chaining, as controls may or may not exist.

Reply:
added